staking/app/StakeConnection.ts

```diff
@@ -80,6 +80,7 @@ export class StakeConnection {
   votingAccountMetadataWasm: any;
   governanceAddress: PublicKey;
   addressLookupTable: PublicKey | undefined;
+  priorityFeeConfig: PriorityFeeConfig | undefined;
```

Comment:
it doesn't need to support undefined when we are setting the default to `{}`

Reply:
you're right